checks/http/broadcom.rb

```diff
@@ -7,16 +7,16 @@ def generate_checks(url)
     [
       {
         :type => "fingerprint",
-        :category => "service",
-        :tags => ["WAF","CORS"],
+        :category => "application",
+        :tags => ["CORS"],
         :vendor => "Broadcom",
         :product => "CA API Gateway",
         :references => ["https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/9-2/learning-center/about-the-ca-api-gateway.html"],
         :version => nil,
         :match_type => :content_headers,
-        :match_content =>  /server: CA-API-Gateway/,
+        :match_content =>  /^server:\ CA-API-Gateway\/[\d\.]{1,}$/i,
```

Comment:
any reason to do '{1,}' vs '+' ?

Reply:
Was thinking about limiting it, like '{1,3}' but then thought to keep it "safe" if they for some reason change the versioning from "9.0" to "9.10"(2 decimal characters for the minor version) and so went with the removal of the second part.

Reply:
Got it, okay. Works for me. '+' is also a viable alternative here, but this works for me